build/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM thinkwhere/gdal-python
+ADD requirements.txt /app/requirements.txt
+# set working directory to /app/
+WORKDIR /app/
+# set up pip and install python packages
+RUN apt-get update &&\
```

Comment:
Checkout the recommendations for `apt-get` here: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Specifically things like doing it all at once, deleting the cache to reduce image size, etc.